### PR TITLE
[TT-849] Tyk-sync 1.2.0 release changes

### PR DESCRIPTION
**Pull request by tbuchaillot**
Added the documentation for --apis and --policies tags for tyk-sync dump command.
Added an example for a single API dump.
Modified dashboard/gateway configurations for tyk-sync.

### CODE REVIEW

**Review by marksou — APPROVED** [review]
LGTM

**Review by matiasinsaurralde — APPROVED** [review]

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/move-policies-between-environments.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Moving policies between two (Dashboard) environments is not as easy as moving AP
 
 ### Preparation
 
-First you must set up your new environment to respect explicit policy IDs. To do so, edit the `tyk.conf` and `tyk_analytics.conf` files in your new environment and set the `policies. allow_explicit_policy_id` setting to `true` (the setting is just `allow_explicit_policy_id` at the root level of the Dashboard configuration).
+First you must set up your new environment to respect explicit policy IDs. To do so, edit the `tyk.conf` and `tyk_analytics.conf` files in your new environment and set the `policies. allow_explicit_policy_id` setting to `true` (the setting is just `allow_explicit_policy_id` at the root level of the Dashboard configuration). Also, if you want to retain your `api_id` you are going to need setting `enable_duplicate_slugs` to `true` in your `tyk_analytics.conf`.
```
Comment (joshblakeley) [inline tyk-docs/content/advanced-configuration/manage-multiple-environments/move-policies-between-environments.md]: This reads a bit oddly.

Something like:

In order to retain your `api_id`  when moving between environments then set `enable_duplicate_slugs` to `true` in your `tyk_analytics.conf`.

Also WHICH tyk_analytics.conf are we talking about. The target dashboard/ origin dashboard/ both?

Try and be really explicit so it is clear how the relationships work

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ dependent tokens continue to have access to your services.
 ### Prerequisites:
 
 - Tyk-Sync was built using Go 1.10. The minimum Go version required to install is 1.7.
+- In order for policy ID matching to work correctly, your Dashboard must have `allow_explicit_policy_id: true` and `enable_duplicate_slugs: true`.
 - In order for policy ID matching to work correctly, your Gateway must have `policies.allow_explicit_policy_id: true`.
```
Comment (joshblakeley) [inline tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md]: Remove duplicate statement/ check for others :)

**Comment by joshblakeley** [comment]
Couple of comments to address before you merge please @tbuchaillot

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md (hunk 2 above)
Comment (tbuchaillot) [reply tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md]: @joshblakeley I don't get what you  meant here

Comment (tbuchaillot) [reply tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md]: nvm

**Comment by tbuchaillot** [comment]
@joshblakeley addressed. Let me know if it's okay 👍

**Comment by tbuchaillot** [comment]
@joshblakeley @marksou any blocker on merging this?

**Comment by joshblakeley** [comment]
Merge away !

**Pull request merged**